.github/labeler.yml

```diff
@@ -0,0 +1,93 @@
+"area: Modem":
+  - "drivers/modem/**/*"
+"area: PWM":
+  - "drivers/pwm/**/*"
+"area: Watchdog":
+  - "drivers/watchdog/**/*"
+"area: Sensors":
+  - "drivers/sensors/**/*"
+"area: ADC":
+  - "drivers/adc/**/*"
+"area: Counter":
+  - "drivers/counter/**/*"
+"area: Timer":
+  - "drivers/timer/**/*"
+"area: I2S":
+  - "drivers/i2s/**/*"
+"area: C Library":
+  - "lib/libc/**/*"
+"area: Deviec Tree":
+  - "dts/**/*"
+  - "**/*.dts"
+"area: I2C":
+  - "drivers/i2c/**/*"
+"area: SPI":
+  - "drivers/spi/**/*"
+"area: Boards":
+  - "boards/**/*"
+"area: POSIX":
+  - "lib/posix/**/*"
+"area: native port":
+  - "arch/posix/**/*"
+  - "soc/posix/**/*"
+  - "**/*native_posix*"
+"area: X86":
+  - "arch/x86/**/*"
+"area: ARM":
+  - "arch/arm/**/*"
+"area: NIOS2":
+  - "arch/nios2/**/*"
+"area: Xtensa":
+  - "arch/xtensa/**/*"
+"area: RISCv32/64":
+  - "arch/risv/**/*"
+"area: ARC":
+  - "arch/arc/**/*"
+"area: Networking":
+  - "subsys/net/**/*"
```

Comment:
we could add here
`- "include/net/**/*"`
`- "include/drivers/ieee802154/**/*"`
`- "drivers/ethernet/**/*"`
`- "drivers/ieee802154/**/*"`
`- "drivers/wifi/**/*"`
`- "drivers/ptp_clock/**/*"`
`- "drivers/net/**/*"`